packages/react/src/drawer-content/DrawerContent.tsx

```diff
@@ -0,0 +1,62 @@
+import * as RadixDrawer from "@radix-ui/react-dialog";
+import { forwardRef } from "react";
+
+import { AnimatePresence } from "../animate-presence";
+import { Backdrop } from "../backdrop";
+import { type BoxProps } from "../box";
+import { useDrawerContext } from "../drawer-context";
+import { Paper } from "../paper";
+import { Transition } from "../transition";
+import * as styles from "./DrawerContent.css";
+
+type DrawerContentProps = BoxProps<
+  typeof RadixDrawer.Content,
+  NonNullable<styles.DrawerVariants>
```

Comment:
Vai, do we need the NonNullable here?

Reply:
Yes - recipe variants can be omitted - meaning they can be `undefined` instead of an object. but that’ll cause issues when merging props - as the final props type can become `any`

Reply:
Then, can we export the type with NonNullable like `ButtonBase.css.ts`, `Alert.css.ts` for consistency over components?